src/Agent/NewRelic/Agent/Core/NewRelic.Agent.Core/AgentManager.cs

```diff
@@ -184,6 +184,12 @@ private void Initialize()
 
             StartServices();
             LogInitialized();
+            LogTlsConfiguration();
+        }
+
+        private void LogTlsConfiguration()
+        {
+            Log.Info($"TLS Configuration (System.Net.ServicePointManager.SecurityProtocol): {System.Net.ServicePointManager.SecurityProtocol}");
```

Comment:
Should we call out that this is the system configuration?
Should this be at the `Debug` level so that it is only being emitted when we suspect there is a Tls issue?

Reply:
I don't think so, as we don't know if people have configured anything... Also, it may not be the system configuration as peoples applications may have changed the security protocol. Also, we should log this at info so it's easy for people and support to see without round tripping with a customer